dev/custom-pages/research.html

```diff
@@ -0,0 +1,176 @@
+<!-- Content -->
+
+</div></div></div>
+<!-- Intro section -->
+<section id="intro-section" data-section-link-title="Research" class="callout callout-intro">
+  <div class="container">
+<div class="row">
+<div class="col-sm-7">
+<h2 class="h1 heading-slab-serif">Research, Partnerships & Innovation</h2>
+<p>UCF researchers drive intellectual capital and innovation to solve today’s most pressing problems. As an emerging preeminent university, we promote economic development in the region by aiding in the transfer of technology and other discoveries between university and industry to create next-generation ideas and products. From nanoscience to optics, from simulation to forensic science, UCF researchers work together to have real-world impact, improving the lives of our students and community. 
+</p>
+<p>
+</div>
+<div class="col-sm-4 col-sm-offset-1">
+
+<a class="ucf-news-thumbnail external" href="http://www.ucf.edu/pegasus/going-public/" target="_blank"><img class="ucf-news-thumbnail-image" style="height: 240px;margin-top: 53px;" src="//wwwqa.cc.ucf.edu/wp-content/uploads/2017/01/going-public-1.png"></a>
+
+</div>
+</div>
+  </div>
+</section>
+<!-- Stats -->
+<section id="stats-section" data-section-link-title="Stats" class="callout callout-stats sections-menu-target homepage-section" style="background-image: url('//www.ucf.edu/wp-content/uploads/2016/12/MainPage-StatsBackgroundnoyellow-1.jpg'); background-position: 50% 50%;">
+  <div class="callout-stats-cover"></div>
+    <div class="container">
```

Comment:
Nitpicky thing, but tabbing is misleading here (.container is not a child of .callout-stats-cover)